Source/WebCore/loader/SubresourceLoader.cpp

```diff
@@ -627,7 +627,7 @@ static void logResourceLoaded(Frame* frame, CachedResource::Type type)
 
 Expected<void, String> SubresourceLoader::checkResponseCrossOriginAccessControl(const ResourceResponse& response)
 {
-    if (!m_resource->isCrossOrigin() || options().mode != FetchOptions::Mode::Cors)
+    if (!m_resource->isCORSCrossOrigin())
```

Comment:
I do not think this is correct here.
We want to have mode CORS and the load to be cross origin.
CachedResource::isCORSCrossOrigin() is basically mode=no-cors and load is cross origin.

Reply:
indeed. I thought the test ended up being the same; but I obviously got that wrong